docs/getting-started.txt

```diff
@@ -107,64 +150,29 @@ might want to review them to gain an understanding of how the Determined API is
 | ``model_def.py`` | Model definition and training/validation loops.                        |
 +------------------+------------------------------------------------------------------------+
 
-***************************************
- Install and Deploy Determined Locally
-***************************************
-
-To install the Determined library and start a cluster locally, enter:
-
-.. code:: bash
-
-   pip install determined
-   det deploy local cluster-up
-
-If your local machine does not have a supported Nvidia GPU, include the no-gpu option to start a
-cluster:
-
-.. code:: bash
-
-   det deploy local cluster-up --no-gpu
-
-**********************************
- Use a Remote Determined Instance
-**********************************
-
-You can also run these examples on a remote Determined master instance by setting the remote IP
-address before running any of the commands:
-
-.. code:: bash
-
-   export DET_MASTER=<ipAddress>:8080
-
-You can also include the master IP address in the command line:
-
-.. code:: bash
-
-   det -m http://<ipAddress>:8080 experiment create distributed.yaml .
+After gaining basic familiarity with Determined tools and operations, you can experiment in your
+domain by replacing these files with your model data and code, and setting configuration parameters
+for the kind of experiments you want to run.
 
 ***********************************
  Run a Single CPU/GPU Training Job
 ***********************************
 
-An experiment is a training job that consists of one or more variations, or trials, of the same
-model. By integrating the Determined API, every experiment automatically gives you metric frequency
-output, plots, and checkpointing without writing extra code. The Determined WebUI displays detailed
-information about the model, the configuration, output logs, and training metrics.
-
 This exercise trains a single model for a fixed number of batches, using constant values for all
-hyperparameters on a single slot.
+hyperparameters on a single *slot*. A slot is a CPU or GPU computing device, which the master
+schedules to run.
 
-#. In the mnist_pytorch directory, create an experiment using the const.yaml configuration file:
+#. In the ``mnist_pytorch`` directory, create an experiment specifying the ``const.yaml``
+   configuration file:
 
    .. code:: bash
 
       det experiment create const.yaml .
 
-   The last argument dot notation (.) uploads all files in the current directory to the Determined
-   cluster and makes them available in the working directory in which each trial is run. This
-   directory is called the context directory.
+   The last dot notation (.) argument uploads all of the example files to the Determined cluster and
+   makes them available in the working, or *context*, directory in which each trial is run.
```

Comment:
I don't think `working directory` and `context directory` are interchangable terms, so `working, or context, directory` seems imprecise.

The context directory is what you upload to the server.  The working directory for training (which is an implementation detail and therefore not documented exactly) is always `/run/determined/workdir`.  We will copy the contents of the context directory you uploaded into your trial's working directory.

So maybe a rephrasing like this would be more appropriate:

The last dot (``.``) argument uploads all of the files in the current directory as the *context directory* for your model.  Determined will copy the contents of the model's context directory into the working directory of the Trial container.